src/Moryx.Resources.Samples/AssembleResource.cs

```diff
@@ -29,6 +29,7 @@ protected override void OnInitialize()
             if (Setup == null)
             {
                 Setup = Graph.Instantiate<AssembleFoo>();
+                Graph.Save(Setup);
```

Comment:
That should not be necessary. Call Changed **must** take care of this.

Reply:
I've fixed that in the ResourceLinker